frontend/src/TodosApp.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import './App.css'
+
+
+ interface Todo{
+  id : string;
+  name: string;
+  date: Date;
+  is_finished: boolean;
+}
+
+function createListItem(name:string, date:Date): Todo{
+  return{
+      id : crypto.randomUUID(),
+      name,
+      date,
+      is_finished: false,
+  }
+}
+
+interface TitlePresenterProps {
+  newMissionName: string;
+  newMissionDate: string;
+  setNewMissionName: (name: string) => void;
+  setNewMissionDate: (date: string) => void;
+  handleAddTodo: () => void;
+  handleDeleteAll: () => void;
+}
+
+function TitlePresenter(props: TitlePresenterProps){
+  return (
+    <div className='todosManager'>
+    <h1>Todo's List</h1>
+      <div id="insert-mission">
+          <input 
+          value={props.newMissionName}
+          type="text"
+          placeholder="Mission Name"
+          onChange={e => props.setNewMissionName(e.target.value)} 
+          />
+          <input
+          value={props.newMissionDate}
+          type="date" 
+          onChange={e => props.setNewMissionDate(e.target.value)} 
+           />
+          <button id="addMission" onClick={() => props.handleAddTodo()}>Add New Mission</button>
+          <button onClick={props.handleDeleteAll}>Delete All Todos</button>
+        </div>
+      </div>
+  )
+}
+
+type onToggle = (id: string) => void
+type onDelete = (id: string) => void
+
+function SingleTodoPresenter(todo: Todo, onToggle: onToggle, onDelete: onDelete){
```

Comment:
SingleTodoPresenter -> TodoPresenter or even TodoItem, its obvious that this this is single because that's its input